File: server.csharp/src/Bananagrams.Dal/Models/Word.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Bananagrams.Dal.Models;

[Table("words")]
public class Word
{
    [Key]
    [Column("id")]
    public int Id { get; set; }
    [Column("description")]
    public string Description { get; set; }
    [Column("image_location")]
    public string ImageLocation { get; set; }
    [Column("title")]
    public string Title { get; set; }
}

File: server.java/bananagrams/src/main/java/coe/bananagrams/entity/Word.java
package coe.bananagrams.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;

@Getter
@Setter
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "words")
public class Word {
    @Id
    @Column(name = "id", updatable = false, nullable = false)
    private int id;
    @Column(name = "description", nullable = false)
    private String description;
    @Column(name = "image_location", nullable = false)
    private String imageLocation;
    @Column(name = "title", nullable = false)
    private String title;
}
